desktop/cmp/contextmenu/ContextMenuSupport.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {defaultMethods} from '@xh/hoist/utils/ClassUtils';
+import {contextMenu} from './ContextMenu';
+import {ContextMenuTarget} from '@xh/hoist/kit/blueprint';
+
+/**
+ * Mixin to allow a Component to show a ContextMenu.
```

Comment:
Would it be helpful to nest within a mixins subdirectory, to separate it a bit from the "component Components"?  We could still export directly out of the index file, as per our standard.